app/src/main/java/com/github/se/wanderpals/MainActivity.kt

```diff
@@ -31,6 +87,34 @@ fun Greeting(name: String, modifier: Modifier = Modifier) {
   Text(text = "Hello $name!", modifier = modifier)
 }
 
+@Composable
```

Comment:
There should be documentation for Google_button function. There is probably documentation convention concerning Kotlin/ Composable that can be followed. Should also use CamelCase naming instead of snake case